docs/getting-started/gs-credentials.md

```diff
@@ -26,14 +26,15 @@ If you're using the integration environment as well as the production environmen
 
```

Comment:
it still mention Prebid.js Express Integration Guide in the table above

Reply:
Thx. Fixed plus did multiple checks and additional fixes.